hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/OpWorkflowMetrics.java

```diff
@@ -77,15 +83,20 @@ public OpWorkflowMetrics(@NonNull final Metrics metrics, @NonNull final ConfigPr
                     .withDescription("The average duration of a " + name + " transaction in nanoseconds")
                     .withUnit("ns");
             final var avgMetric = metrics.getOrCreate(avgConfig);
-            transactionMetrics.put(functionality, new TransactionMetric(maxMetric, avgMetric));
+            transactionDurationMetrics.put(functionality, new TransactionMetric(maxMetric, avgMetric));
+
+            // initialize the transaction throttle metrics
+            final var throttledConfig = new Counter.Config("app", name + "ThrottledTps")
+                    .withDescription("The number of " + name + " transactions that failed due to throttling");
```

Comment:
Just a suggestion for precision
```suggestion
            final var throttledConfig = new Counter.Config("app", name + "ThrottledTxns")
                    .withDescription("The number of " + name + " transactions that were throttled");
```

Reply:
I agree with the "ThrottledTps" -> "ThrottledTxns" change, thank you.
For the "transactions that failed due to throttling" -> "transactions that were throttled" change, I disagree. I think that "transactions that were throttled" sounds like we're counting the txs that are subject to throttling (i.e. all the transactions received that don't fail due to ingest checks), not just the txs that we rejected due to throttle limits.

Reply:
I updated this, see if you like my wording @mhess-swl

Reply:
Looks great, thanks!